### DISCUSSION TITLE
Cluster managment

### DISCUSSION THREAD

**Discussion by I-dev-code** [gravitational/teleport#28418] [Q&A]
Hello everyone, 

So i've started using teleport since a week, and i was wondering how to join another cluster to teleport.  On the doc i saw the trusted cluster, but i'm don't think that's what i need.
Trusted cluster allow teleport to connect multiple teleport service with one root teleport connecting them all. 
But i'd like to keep one instance of teleport and manage clusters  into it. 
Tell me if it's not clear i'll try to explain better.

So i tried to create a new yml with "teleport configure" but of course when i start it, there is already a running config.
I tried  to join a cluster to /etc/teleport.yaml and creat a new data_dir but i doesn't work. Is it possible or do i need to do trusted cluster? 
thanks guys for the time.

**Comment by I-dev-code** [comment]
Ps: There is also a way to manage multiple cluster with k8s but is it mandatory ?

**Comment by webvictim** [comment]
What do you define as a "cluster" here? You can run one Teleport cluster and connect one or more of any of these to it:
- SSH nodes
- Kubernetes clusters
- Applications
- Databases
- Windows desktop services

If you want to link multiple Teleport clusters in different logical locations for you, that's when you'd want trusted clusters.
